project-resources/GITHUB_REQUESTS/2023.md

```diff
@@ -2,3 +2,4 @@
 
 First Name | Last Name | Project | Amount requested | Pull Request date | Pull Request link
 -|-|-|-|-|-
+Claudio | Wunder | Node.js | 100.00 USD | 2023.08.16 | https://github.com/openjs-foundation/cross-project-council/pulls/1149 |
```

Comment:
```suggestion
Project  | Request | Cost / Period | Date | Champion | Pull Request link
-|-|-|-|-|-
Node.js | Codespaces | USD 100 / month | 2023.08.16 | Claudio Wunder | https://github.com/openjs-foundation/cross-project-council/pulls/1149 |
```

A few suggestions about the format of the requests in general (as this is the first one):

- I initially saw all of those zeroes and thought you were asking for a lot more; we don’t need to mention cents here.
- Knowing that this is a recurring expense is important. We could even consider a `frequency` column.
- `first name` / `last name` columns makes this look like a personal expense, so maybe we should use `project` / `champion` instead.
-  It's valuable to know what the request is for inline.

Reply:
This should probably be addressed on a separate PR 👀

Reply:
> This should probably be addressed on a separate PR 👀 (that should be merged before this one)

Whatever feels more convenient to you. I'm fine either way.

Reply:
Sure. Imma gonna proceed and open another PR for this + some changes to remove the cents mention on the template and also mention that these costs are intended to be recurring

Reply:
PR: https://github.com/openjs-foundation/cross-project-council/pull/1150